cpp/include/cudf/table/experimental/row_operators.cuh

```diff
@@ -858,7 +1001,7 @@ class device_row_comparator {
   table_device_view const rhs;
   Nullate const check_nulls;
   null_equality const nulls_are_equal;
-};
+};  // class device_row_comparator
```

Comment:
IDE magic? I think I'd revert this.
```suggestion
};
```

Reply:
If it's consistent with the rest of the file, feel free to keep it. I wasn't aware of that pattern.

Reply:
Oops, missed your followup -- it's fine both ways....